src/router/BaseRouter.php

```diff
@@ -14,6 +14,10 @@
 use function Facebook\AutoloadMap\Generated\is_dev;
 
 abstract class BaseRouter<+TResponder> {
+  public function __construct(
+    private BaseRouterOptions $options = shape('use_get_responder_for_head' => true)
```

Comment:
Perhaps `?BaseRouterOptions`, and make all the fields optional?

This will be needed when a second option is added - it doesn't really matter now, but doing it would make it clearer what the 'right way' is to add one.

This would also need something like:

```Hack
$this->options = shape(
  'use_get_responder_for_head' => $options['use_get_responder_for_head'] ?? true
);
```